docs/metrics.md

```diff
@@ -0,0 +1,13 @@
+# Cluster Network Addons Operator metrics
```

Comment:
maybe worth please to add an header that this file is auto generated by metricsdocs and should not changed manually

something like 
This file is auto generated by metricsdocs, run make generate-doc in order to update it.

And then line 13 can be removed
(the tool should be updated to do this changes of course)

Reply:
updated